### ISSUE TITLE
Add feature to automatically pick .env file(s) from project_root directory

### ISSUE THREAD

**Issue by mtahirtariq** [Ashald/EnvFile#31]
Thanks for this awesome project, it solved a lot of issues with env management for me :).
I believe it would be awesome to add a feature (may be disabled by default) which when enabled make `EnvFile` to automatically load `.env` file(s) from `project_root`.
So that we enable this feature and we don't need to add `.env` files in run configuration for each project we just place the `.env` file in `project_root` and it will automatically be loaded.

**Comment by ashald** [comment]
At the first glance this sounds as a cool idea but on a 2nd thought I'm wondering whether this is really a good thing to do. After all, there are too many questions and unclarity. What if there are several `.env` files? It might be non-intuitive where variables are coming from and so on. I believe you can try to achieve similar functionality by editing the run configuration template you use. If that sounds good for you please give it a try and let me know how it worked.

**Comment by ashald** [comment]
Any feedback on proposed workaround with run configuration templates?

**Comment by iddan** [comment]
The common practice is to only load a file with the exact name `.env`. Other `.env` files should be consumed explicitly

**Comment by ashald** [comment]
Does the workaround proposed here works to enable the desired behavior https://github.com/Ashald/EnvFile/issues/55#issuecomment-371002257 ?

**Comment by iddan** [comment]
No, because you can't refer to a `.env` file relatively: If you'd allow using `%PROJECT_ROOT%` variable it will be possible.

**Comment by iddan** [comment]
Yeah, it's more of a matter of defaults.

**Comment by aotd1** [comment]
It can be achieved using the IDE notification dialog.
If .env file exists in project, plugin can propose popup dialog to automatically add it in all / specific run configuration (s).

**Comment by devurandom** [comment]
> I believe you can try to achieve similar functionality by editing the run configuration template you use. If that sounds good for you please give it a try and let me know how it worked.

Thanks for the hint! I tried it (https://www.jetbrains.com/help/idea/run-debug-configuration.html#templates explains where to find the config) and it works very well. I especially like that (after you have figured out how to do it) it is very obvious which variables from which files are loaded where -- no automagic. :)

**Comment by bodograumann** [comment]
It seems the workaround from #55  does not work anymore :-( @ashald 
I've added the env file to the template, but new run configurations don't have it enabled.
